lib/galaxy/tools/stock.py

```diff
@@ -26,7 +26,7 @@ def stock_tool_sources():
 
 
 def _walk_directory_for_tools(path):
-    if path.is_file() and path.name.endswith(".xml"):
+    if path.is_file() and path.name.endswith(".xml") and "macros" not in path.name and "_conf.xml" not in path.name:
```

Comment:
can you add a comment what you are trying here?

Reply:
Added a comment.